docs/conf.py

```diff
@@ -20,18 +17,14 @@
 author = f"{project} Contributors"
 copyright = f"The {author}"
 
-# The short X.Y version
-version = ".".join(
-    get_scm_version(
-        local_scheme="no-local-version",
-    ).split(
-        "."
-    )[:3],
-)
-
 # The full version, including alpha/beta/rc tags
-release = get_scm_version()
+release = get_version(project)
+
+# The short X.Y version
+version = ".".join(release.split(".")[:3])
 
+print("pip-tools version:", version)
+print("pip-tools release:", release)
```

Comment:
Accidentally left over?

Reply:
Left on purpose to see versions in logs:
```
$ python -m sphinx ...
Running Sphinx v6.2.1
pip-tools version: 7.0.1
pip-tools release: 7.0.1.dev4
making output directory... done
...
```

Reply:
Maybe a comment about that could be useful?

Reply:
Addressed in a49c505c435d3e7e42bd2f0d8ed7ebdbd2701d42.